.storybook/locales.js

```diff
@@ -0,0 +1,58 @@
+import * as am from '../app/_locales/am/messages.json';
```

Comment:
TIL that you can't `export * as ___ from ...` yet. [Soon hopefully](https://github.com/tc39/proposal-export-ns-from)!